cocos2d/core/sprites/CCSGSpriteWebGLRenderCmd.js

```diff
@@ -23,18 +23,23 @@
  ****************************************************************************/
 
 var macro = cc.macro;
-var ccgl = cc.gl;
 
 //Sprite's WebGL render command
 _ccsg.Sprite.WebGLRenderCmd = function (renderable) {
     _ccsg.Node.WebGLRenderCmd.call(this, renderable);
     this._needDraw = true;
 
-    this._quad = new cc.V3F_C4B_T2F_Quad();
-    this._quadWebBuffer = cc._renderContext.createBuffer();
-    this._quadDirty = true;
+    this._vertices = [
+        {x: 0, y: 0, u: 0, v: 0}, // tl
+        {x: 0, y: 0, u: 0, v: 0}, // bl
+        {x: 0, y: 0, u: 0, v: 0}, // tr
+        {x: 0, y: 0, u: 0, v: 0}  // br
+    ];
```

Comment:
Use flat data structure instead of V3F_C4B_T2F quad, it's much faster to read&write and use less memories.